src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SearchBundle\Controller;
+
+use Massive\Bundle\SearchBundle\Search\SearchManagerInterface;
+use Sulu\Bundle\WebsiteBundle\Resolver\ParameterResolverInterface;
+use Sulu\Component\Rest\RequestParametersTrait;
+use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * This controller handles the search for the website.
+ */
+class WebsiteSearchController
+{
+    use RequestParametersTrait;
+
+    /**
+     * @var SearchManagerInterface
+     */
+    private $searchManager;
+
+    /**
+     * @var RequestAnalyzerInterface
+     */
+    private $requestAnalyzer;
+
+    /**
+     * @var ParameterResolverInterface
+     */
+    private $parameterResolver;
+
+    /**
+     * @var EngineInterface
+     */
+    private $engine;
+
+    public function __construct(
```

Comment:
Adding it, because I think we said that we do that once. Although I am not a big fan of it, especially in this case, since it doesn't add any value.